quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testBuildHelp.unix.approved.txt

```diff
@@ -44,13 +44,15 @@ Transaction:
 
 Feature:
 
---features <feature> Enables a set of one or more features. Possible values are: account-api,
-                       account2, account3, admin-api, admin-fine-grained-authz, admin2,
-                       authorization, ciba, client-policies, client-secret-rotation,
-                       declarative-user-profile, device-flow, docker, dpop, dynamic-scopes, fips,
-                       impersonation, js-adapter, kerberos, linkedin-oauth, multi-site, par,
-                       preview, recovery-codes, scripts, step-up-authentication, token-exchange,
-                       transient-users, update-email, web-authn.
+--features <feature> Enables a set of one or more features. Possible values are: account-api[:v1],
```

Comment:
The same as I've mentioned in the [FeaturesDistTest.java](https://github.com/keycloak/keycloak/pull/24811/files#diff-8ed21c4774db2d7e039cbd429ff266dcdb1782ff4f74ef0196ac27686c139c26)